app/src/pages/playground/schemas.ts

```diff
@@ -93,11 +95,80 @@ const chatMessageSchema = schemaForType<ChatMessage>()(
 export const chatMessagesSchema = z.array(chatMessageSchema);
 
 /**
- * The zod schema for llm model name
+ * The zod schema for llm model config
  * @see {@link https://github.com/Arize-ai/openinference/blob/main/spec/semantic_conventions.md|Semantic Conventions}
  */
-export const modelNameSchema = z.object({
+export const modelConfigSchema = z.object({
   [SemanticAttributePrefixes.llm]: z.object({
     [LLMAttributePostfixes.model_name]: z.string(),
+    [LLMAttributePostfixes.invocation_parameters]: z
+      .string()
+      .transform((s) =>
+        z
+          // TODO(apowell): specify schema for parameters that we care about
```

Comment:
ticket here

Reply:
just want to make sure that this won't fail and drop the model name if the invocation params are off, if so it might be worth parsing twice with two different schemas just so that we get as much information as we can. Let me know if this doesn't make sense and we can take a look together

Reply:
can we use your schema below with passthrough?

Reply:
I think so

Reply:
Okay so this turned out to be awesome. We can now read invocation params from the span and convert them into the expected format automatically, including snake cased keys.

Reply:
still want to make sure we can get llm model name and stuff even if invocation params are busted. Can also just not use zod for model name and just traverse to it if that makes it easier

Reply:
Fixed